### PR TITLE
OTLP: Remove experimental word form OTLP receiver

### PR DESCRIPTION
**Pull request by jesusvazquez**
The OTLP receiver can now considered stable. We've had it for longer than a year in main and has received constant improvements.

### CODE REVIEW

File: cmd/prometheus/main.go [hunk 1]
```diff
@@ -187,7 +187,7 @@ func (c *flagConfig) setFeatureListOptions(logger log.Logger) error {
 				level.Warn(logger).Log("msg", "Remote write receiver enabled via feature flag remote-write-receiver. This is DEPRECATED. Use --web.enable-remote-write-receiver.")
 			case "otlp-write-receiver":
 				c.web.EnableOTLPWriteReceiver = true
-				level.Info(logger).Log("msg", "Experimental OTLP write receiver enabled")
```
[the remark this answers is not in the document]
Comment (arukiidou) [reply cmd/prometheus/main.go:190]: - I think #10119 is very similar to this PR.

File: cmd/prometheus/main.go [hunk 2]
```diff
@@ -345,6 +342,9 @@ func main() {
 	a.Flag("web.remote-write-receiver.accepted-protobuf-messages", fmt.Sprintf("List of the remote write protobuf messages to accept when receiving the remote writes. Supported values: %v", supportedRemoteWriteProtoMsgs.String())).
 		Default(supportedRemoteWriteProtoMsgs.Strings()...).SetValue(rwProtoMsgFlagValue(&cfg.web.AcceptRemoteWriteProtoMsgs))
 
+	a.Flag("web.enable-otlp-receiver", "Enable API endpoint accepting OTLP write requests.").
```
Comment (ArthurSens) [inline cmd/prometheus/main.go:345]:
```suggestion
	a.Flag("web.otlp-receiver", "Enable API endpoint accepting OTLP write requests.").
```

Just to be consistent with remote-write. The flag there is `web.remote-write-receiver`, right?

Comment (jesusvazquez) [reply cmd/prometheus/main.go:345]: Yes it already is, check https://github.com/prometheus/prometheus/pull/14894/files#diff-6554f0eaaf28d34fad1ec0ad13393db25deb69c649b377839173b87c946aae9cR336-R339

Comment (ArthurSens) [reply cmd/prometheus/main.go:345]: ah, sorry about that... I don't know why the remote write flag was called `web.remote-write-receiver` 🤨

**Review by ArthurSens — APPROVED** [review]

File: cmd/prometheus/main.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jesusvazquez) [reply cmd/prometheus/main.go:187]: Thanks, addressed this suggestions.

**Pull request merged**